### PR TITLE
make style configurable for commit command

### PR DESCRIPTION
**Pull request by Lee-W**
#37

### CODE REVIEW

File: commitizen/cz/base.py [hunk 1]
```diff
@@ -1,13 +1,29 @@
...
+from typing import Optional, List, Tuple
 from abc import ABCMeta, abstractmethod
 
+from prompt_toolkit.styles import merge_styles, Style
+
 
 class BaseCommitizen(metaclass=ABCMeta):
     bump_pattern: Optional[str] = None
     bump_map: Optional[dict] = None
+    DEFAULT_STYLE_CONFIG: List[Tuple[str, str]] = [
```
Comment (Woile) [inline commitizen/cz/base.py]: What do you think of making this lower case to be consistent with the other attributes?

**Comment by Woile** [comment]
Looks good @Lee-W ! Thanks a lot!

File: commitizen/cz/base.py [hunk 2]
```diff
@@ -17,6 +33,10 @@ def questions(self) -> list:
     def message(self, answers: dict) -> str:
         """Format your git message."""
 
+    @property
+    def style(self):
+        return merge_styles(BaseCommitizen.DEFAULT_STYLE_CONFIG, Style(self.config["style"]))
```
Comment (Woile) [inline commitizen/cz/base.py]: This line fails, I think it should be something like this instead:

```python
@property
def style(self):
    return merge_styles(
        [Style(BaseCommitizen.DEFAULT_STYLE_CONFIG), Style(self.config["style"])]
    )


```

File: commitizen/cz/base.py [hunk 3]
```diff
@@ -1,13 +1,29 @@
...
+from prompt_toolkit.styles import merge_styles, Style
+
 
 class BaseCommitizen(metaclass=ABCMeta):
     bump_pattern: Optional[str] = None
     bump_map: Optional[dict] = None
+    DEFAULT_STYLE_CONFIG: List[Tuple[str, str]] = [
+        ("qmark", "fg:#673ab7 bold"),
+        ("question", "bold"),
+        ("answer", "fg:#f44336 bold"),
```
Comment (Woile) [inline commitizen/cz/base.py]: I would use `#FF9D00` or `#673ab7` for the answer. Whichever you choose, it should also be the same for: `qmark`, `pointer`, `highlighted`. What do you think?

File: commitizen/cz/base.py (hunk 1 above)
Comment (Lee-W) [reply commitizen/cz/base.py]: I agree with you. I've updated it. Thanks!

File: commitizen/cz/base.py (hunk 2 above)
Comment (Lee-W) [reply commitizen/cz/base.py]: Yes, I've fixed it. Thanks!

File: commitizen/cz/base.py (hunk 3 above)
Comment (Lee-W) [reply commitizen/cz/base.py]: I have no preference on colors. I just change it to `#FF9D00`

**Comment by Woile** [comment]
Great work @Lee-W ! gonna release soon :tada:

**Pull request merged**